apps/web/src/pages/ApplicantDashboardPage/ApplicantDashboardPage.tsx

```diff
@@ -43,6 +71,54 @@ export const DashboardPage = ({ currentUser }: DashboardPageProps) => {
         )}
         subtitle={""}
       />
+      <section data-h2-margin="base(x3, 0)">
+        <div data-h2-wrapper="base(center, large, x1) p-tablet(center, large, x2)">
+          <div
+            data-h2-display="base(flex)"
+            data-h2-flex-direction="base(column) p-tablet(row)"
+            data-h2-gap="base(x1)"
+          >
+            {isVerifiedEmployee ? (
+              <CareerDevelopmentTaskCard
+                moveInterests={moveInterestsMapped}
+                organizationTypeInterests={organizationTypeInterestsMapped}
+                communityInterestsQuery={communityInterestsQuery}
+              ></CareerDevelopmentTaskCard>
```

Comment:
This can be self closing `<CareerDevelopmentTaskCard />` since there are no children. I noticed other examples of this throughout as well.

Reply:
Sure, sure. https://github.com/GCTC-NTGC/gc-digital-talent/pull/12607/commits/a8c03b88b437247fadf0ba42c42446177f67a2c1
Should this be something we automatically lint for 🤔

Reply:
I think maybe? Honestly it is mostly me nit picking :laughing: I don't think there is anything wrong with it, we just don't do it very often :stuck_out_tongue: